tests/foreman/api/test_oscappolicy.py

```diff
@@ -76,14 +76,14 @@ def test_positive_crud_scap_policy(
         assert policy.deploy_by == 'puppet'
         assert policy.name == name
         assert policy.description == description
-        assert policy.scap_content_id == scap_content["scap_id"]
-        assert policy.scap_content_profile_id == scap_content["scap_profile_id"]
-        assert policy.tailoring_file_id == tailoring_file["tailoring_file_id"]
+        assert str(policy.scap_content_id) == str(scap_content["scap_id"])
```

Comment:
This comparison was failing after I fixed the fixture, so I wrapped all of the ID comparisons here to make them consistent types.